### ISSUE TITLE
Horrible typing issues with a big table in a long document

### ISSUE THREAD

**Issue by andupaz** [Zettlr/Zettlr#4657]
### Description

I'm writing my thesis, it's therefore a long document (right now around 46k characters) and wanted to introduce a table. After reading the table editor instructions after a few failed tries (sorry, not a very intuitive feature), I created a table of around 8 rows and 4 columns. As soon as it was saved, and I wanted to start typing again, the typing was EXTREMELY slow and also did not type correctly - as it was catching every other typed letter.

### Reproducing

Create a long document (mine is around 45k characters) and a large table and start writing. Not sure if reproducible or just me.

### Zettlr Version

Stable (most recent version)

### Installation Method

None

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 11

**Comment by nathanlesage** [comment]
This could be a duplicate of #4604. The issue owner already sent me their file and I will at some point get to investigating what is causing the issue, maybe this will solve your issue as well.

**Comment by andupaz** [comment]
Amazing, thank you for the quick response!

**Comment by nathanlesage** [comment]
Quick update: I have now improved performance of a series of computations by large margins, and the other document is lagging much less – maybe it is now also better for you? To test this, please wait a few hours until a nightly build that I just scheduled is finished. Then, download the nightly release, install it, and see if the performance is better with it.

Here's the link, you'll know that the build is complete when the "last modification time" is being updated. https://nightly.zettlr.com

**Comment by andupaz** [comment]
Thanks! Can I rely on this nightly release without worrying about potential data loss? I’m writing my thesis and would therefore like to avoid any unnecessary hassles.

**Comment by nathanlesage** [comment]
Yes, I haven't changed much since the 3.0.0 release -- at least nothing that might lead to data loss, so you should be safe! I'll release the changes so far anyhow this week.
